Added localization

nitricware opened the pull request:
Hy!

I added localization support to your already great template engine.
Hope you like and merge it!

Thanks,
Kurt

nitricware committed:
added localization support

tonic now saves the contents of an XML file as a localized array which
then can be accessed easily like any other array in the template

nitricware committed:
updated readme

nitricware committed:
added localization support for multiple files

You can now use multiple xml files to localize your php application.

nitricware committed:
support for multiple localization files

- support for multiple localization files
- readme update
- composer update

Commit:
Merge conflicts resolved

Commit:
Merge branch 'rgamba-master'

Localization Support and Update to 3.1

adaniello commented:
@nitricware I think that localization should be keep out of a template library: everyone can choose the way

Commit:
Bugfix

Fixed a bug where 'or' in a word (i.e. 'sorts') was interpreted as an operator. (Also fixes 'eq', 'and', 'gt', ...)

nitricware committed:
NitricWare Fork

Removed unneccessary files

nitricware committed:
Added auto print on render

Added a feature to print templates on render.

nitricware committed:
PHP 7 Fixes

nitricware committed:
dockerzied, added basic example for docker test

nitricware committed:
mini example fix

nitricware committed:
added type to fields where known

nitricware committed:
added types

nitricware committed:
fix: added types

nitricware committed:
Finalized PHP 7.4 Update

1. Added supression tags to doc block
2. removed unused functionality ($is_php)
3. added type hinting

nitricware committed:
updated Readme for Tonic 3.3

Signed-off-by: Kurt Höblinger <office@nitricware.com>

nitricware committed:
added to composer

nitricware committed:
updated composer.json

nitricware committed:
updated version string

nitricware committed:
updated readme

nitricware committed:
composer fixes

nitricware committed:
updated version

nitricware committed:
composer fix

nitricware committed:
readme update
